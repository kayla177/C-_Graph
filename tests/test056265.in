ENTITY R91XA6BF name_R91X journal
ENTITY RGEBLT7B name_RGEB paper
ENTITY YUHR14OZ name_YUHR journal
ENTITY GKWRM3Z4 name_GKWR journal
ENTITY WCIAXR95 name_WCIA paper
ENTITY SX2BZUAL name_SX2B conference
ENTITY MGE70TJG name_MGE7 paper
ENTITY NVAJTTMR name_NVAJ conference
ENTITY RVC99DHY name_RVC9 conference
ENTITY 9ALDMF59 name_9ALD author
RELATIONSHIP YUHR14OZ published_in MGE70TJG 2.0
RELATIONSHIP YUHR14OZ affiliated_with 9ALDMF59 1.2
RELATIONSHIP NVAJTTMR authored_by 9ALDMF59 1.3
RELATIONSHIP SX2BZUAL affiliated_with RVC99DHY 2.9
RELATIONSHIP R91XA6BF cited_by NVAJTTMR 0.5
RELATIONSHIP RVC99DHY collaborated_with RGEBLT7B 0.4
RELATIONSHIP RGEBLT7B presented_at GKWRM3Z4 6.5
RELATIONSHIP RVC99DHY published_in WCIAXR95 9.3
RELATIONSHIP YUHR14OZ cited_by RGEBLT7B 9.5
RELATIONSHIP SX2BZUAL presented_at RGEBLT7B 4.9
RELATIONSHIP RVC99DHY collaborated_with MGE70TJG 6.5
RELATIONSHIP R91XA6BF authored_by 9ALDMF59 2.3
RELATIONSHIP GKWRM3Z4 authored_by R91XA6BF 3.8
RELATIONSHIP YUHR14OZ collaborated_with RVC99DHY 0.9
RELATIONSHIP SX2BZUAL collaborated_with 9ALDMF59 1.3
PRINT RGEBLT7B
PATH SX2BZUAL MGE70TJG
DELETE NVAJTTMR
PRINT R91XA6BF
DELETE MGE70TJG
PATH R91XA6BF 9ALDMF59
DELETE RVC99DHY
PRINT WCIAXR95
HIGHEST
DELETE GKWRM3Z4
EXIT
