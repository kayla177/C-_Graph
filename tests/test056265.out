success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
RVC99DHY GKWRM3Z4 YUHR14OZ SX2BZUAL
SX2BZUAL RVC99DHY MGE70TJG 9.4
success
9ALDMF59 GKWRM3Z4
success
R91XA6BF 9ALDMF59 2.3
success

R91XA6BF YUHR14OZ 19.8
success
